api/capi/include/nntrainer.h

```diff
@@ -144,6 +144,20 @@ int ml_nnmodel_train_with_generator(ml_nnmodel_h model,
  */
 int ml_nnmodel_destruct(ml_nnmodel_h model);
 
+/**
+ * @brief Get the summary of the neural network model.
+ * @details Use this function to get the summary of the neural network model.
```

Comment:
Lets also add model training, validation and testing results from the previous run, if any in this summary.

Reply:
I made an issue for that #290

Reply:
I think #290 is different. I meant that we can add performance metrics in model summary about a previous training/validation/testing etc

Reply:
Oh.. There are two kinds of summary:

1. summary to get the (current struct) of the model (equivalent to `model.summary()` in keras)
2. get the performance score (equivalent to `model.evaluate()` in keras)

I was more focusing on 1.

can nntrainer able to support 2. now?

Reply:
Training in nntrainer supports running validation and testing. However, there is no way to report its result.
Let's add test/valid loss in there, and later, there can be a proper metric for the results.

Reply:
Alright